### ISSUE TITLE
Include Filename in Errors

### ISSUE THREAD

**Issue by beeing** [javascript-obfuscator/javascript-obfuscator#513]
When there is error and there are hundreds of files, it is time-consuming to find out which file causes the problem.

It will help save time to find out the offending file so we can zoom in to which line that causes it.

Thanks!

**Comment by sanex3339** [comment]
Nice idea

**Comment by sanex3339** [comment]
https://github.com/javascript-obfuscator/javascript-obfuscator/pull/515
Will out in `0.24.0` later this week

**Comment by sanex3339** [comment]
Released as `0.24.0`

**Comment by beeing** [comment]
Hi @sanex3339, just tested this out and found the below output:

```
Error: ERROR in dist, line 421: Identifier 'a15a' has already been declared (421:6)
```

The `dist` is the directory name, however the path to file is missing.

**Comment by sanex3339** [comment]
File path was added to parsing errors that have a format like:
```
/ERROR in \/src\/foo\.js, line 3: Unexpected token \(3:32\)\n.*\.\.\.var baz = 3;,\.\.\./
```

**Comment by sanex3339** [comment]
How did you use it?
I tried just to obfuscate single .js file with the CLI and got:
```
Error: ERROR in foo.js, line 2: Identifier 'foo' has already been declared (2:4)
```

**Comment by sanex3339** [comment]
Ah, i reproduced it.

**Comment by sanex3339** [comment]
Released as `0.24.6`
